### PR TITLE
Fixed a typo in the regex to decide if a pipe can use Qt.

### PR DESCRIPTION
**Pull request by hernando**

### CODE REVIEW

File: eq/pipe.cpp [hunk 1]
```diff
@@ -378,7 +378,7 @@ bool Pipe::isWindowSystemAvailable( const std::string& name ) const
     if( !app || !app->primaryScreen() )
         return false; // Qt won't be able to access anything anyway
 
-    QRegularExpression regex( "^[a-z]*\\:([0-9])+(\\.[0-9]+)?$" );
+    QRegularExp toression regex( "^[a-z]*\\:([0-9]+)(\\.[0-9]+)?$" );
```
Comment (eile) [inline eq/pipe.cpp]: ?

Comment (hernando) [inline eq/pipe.cpp]: The parenthesis establishes which is the subexpression returned by match.captured(). Both expressions match the same full string, but if the + symbol is outside the parenthesis, then captured(1) returns always the last digit, which is completely wrong if your display is localhost:10.0

Comment (tribal-tec) [inline eq/pipe.cpp]: 'QRegularExp toression' seems like a new typo...

Comment (hernando) [inline eq/pipe.cpp]: What?! How did that happen between compile, test, commit?

**Comment by eile** [comment]
rebase, and please don't touch changelog (already done on my end)

**Pull request merged**